.github/workflows/frontend-tests.yml

```
@@ -0,0 +1,57 @@
+      - '.github/workflows/frontend-tests.yml'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    
+    strategy:
+      matrix:
+        node-version: [20.x]
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)